src/twisted/words/test/test_service.py

```diff
@@ -212,7 +212,7 @@ def _response(self, user, messageType=None):
         If messageType is defined, only messages of that type will be returned.
         """
         response = user.transport.value()
-        if bytes != str and isinstance(response, bytes):
+        if isinstance(response, bytes):
```

Comment:
According to the test coverage report,
I think that this line can be removed and the test will still pass :)